### ISSUE TITLE
[A11y] Alt Text Field Population Broken (Staging)

### ISSUE THREAD

**Issue by coleherman43** [OregonDigital/OD2#3699]
### Operating System

Windows

### Browser

Firefox

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

Alt text derived from an image's metadata does not appear in the ['alt' tag](https://www.w3schools.com/TAGS/att_alt.asp) when inspecting image properties in the work viewer. Across 2 different imports and file types (tiff and png) the alt tag has "file details" rather than the actual image alt text. 

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/e7df4905-76e5-4bd9-b7e9-a9e7680666bb" />

Inspecting the accessibility properties shows that the associated value for the image is also "file details", while the real alt text appears conjoined with the file ID in the "rowheader" field.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/17596c54-130d-4466-bc4b-f3a389e07d64" />

### Suggested Fix

Ensure that image properties are correctly transferred from file data to html tags.

### Can anybody QA or does this need an expert?

Anybody

### QA Steps

Go to works show page (either during approval or in publicly viewable version) and click inspect on any images with alt text attached. Check that the "alt" tag of the image matches the "Alternative Text" field of the work page. Then right click on the image and inspect accessibility properties. and check that the name associated with the image also matches the "Alternative Text" field.

**Comment by briesenberg07** [comment]
some hopefully helpful additional information from [8/4 metadata meeting](https://docs.google.com/document/d/1y6eeZjhE3mpziQ1Xk6KXMhZbMpxHVWmHy3R6XqW9vzY/edit?tab=t.0#bookmark=id.84r309a9o1o2)\*

confirmed with @wickr that the `img alt` attribute with unhelpful "file details" value corresponds to the file set thumbnail displayed at bottom of show page in WORKS AND FILES. So for example work in staging [4m90dv62j](https://staging.oregondigital.org/concern/images/4m90dv62j?locale=en), looking at HTML source...

```HTML
<td class="attribute attribute-thumbnail">
      <a href="/concern/file_sets/f04m90dv62j?locale=en"><img alt="file details" src="/downloads/f04m90dv62j?file=thumbnail" /></a>
</td>
```

...corresponding to file set thumbnail in UI below.

<img width="1002" height="458" alt="the WORKS AND FILES section of an Oregon Digital work showpage, displaying a thumbnail for a .png file set" src="https://github.com/user-attachments/assets/6634f2ef-5ab4-4a26-a07c-5fb966f2d2e7" />

  
Unclear to metadata team whether/how difficult to pull the file-set level alt text value through here. We also noted that the initial scope of work for image file set alt text functionality didn't include any details about whether/how the data might come through in page source HTML, so unclear whether this would be considered in scope for the original work on #3637 .

Also seems relevant that alt text value does seem to be coming through in HTML source near the `img` element for the thumbnail -- see 

```HTML
<td class="attribute attribute-thumbnail">
      <a href="/concern/file_sets/f04m90dv62j?locale=en"><img alt="file details" src="/downloads/f04m90dv62j?file=thumbnail" /></a>
      <!-- img alt for thumbnail in WORKS AND FILES -->
    </td>
    <th scope="row" class="attribute attribute-filename ensure-wrapped"><a href="/concern/parent/4m90dv62j/file_sets/f04m90dv62j">this_is_art_02.png</a> <br/><br/>
          <span class="badge badge-info">Accessible Copy</span> <br/>
        Accessibility Features: Alternative Text
          <br/><br/>
          <span aria-label="Alternative Text">Alternative Text:</span> <span class="alt-text-display">THIS IS ART in red letters on a black background, crossed out with a translucent green X</span>
          <!-- the alt text value does appear in this span -->
    </td>
```
  

\*_not publically accessible_

**Comment by briesenberg07** [comment]
**QA pass** 👍🏻 
Details from checks on `staging.oregondigital.org`

- [x] work [j098zb23s](https://staging.oregondigital.org/concern/images/j098zb23s) with one file set: **pass**, details below
- [x] work [6m311p41f](https://staging.oregondigital.org/concern/images/6m311p41f?locale=en) with multiple (2) file sets: **pass**, details below
- [x] work [t148fh19g](https://staging.oregondigital.org/concern/images/t148fh19g?locale=en) which is part of complex object [fq977t82f](https://staging.oregondigital.org/concern/generics/fq977t82f?locale=en): **pass**, details below

```HTML
<!-- work j098zb23s snippet -->
    <td class="attribute attribute-thumbnail">
      <a href="/concern/file_sets/f0j098zb23s?locale=en"><img alt="Straight, circular, and curved lines in green and black on a gray background" src="/downloads/f0j098zb23s?file=thumbnail" /></a>
    </td>

<!-- work 6m311p41f snippets -->
    <td class="attribute attribute-thumbnail">
      <a href="/concern/file_sets/f06m311p41f?locale=en"><img alt="a stylized image of a tent, clouds, a colorful sky, a river, and musrooms, with the word Oregon in a hand-drawn font" src="/downloads/f06m311p41f?file=thumbnail" /></a>
    </td>
    <!-- ... -->
    <td class="attribute attribute-thumbnail">
      <a href="/concern/file_sets/f16m311p41f?locale=en"><img alt="The back of a postcard – includes the title “Oregon – Groovy Escape – Camping”" src="/downloads/f16m311p41f?file=thumbnail" /></a>
    </td>

<!-- work t148fh19g snippet -->
    <td class="attribute attribute-thumbnail">
      <a href="/concern/file_sets/f0t148fh19g?locale=en"><img alt="a close-up of a nametag with a name, pronouns, and place of work -- the nametag is pinned to a green fleece garment" src="/downloads/f0t148fh19g?file=thumbnail" /></a>
    </td>
```
